src/tests/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ status_test.cxx
 status_writer.cxx
 command_handler.cxx
 stream_master.cxx
+MainOptTest.cpp
```

Comment:
Since we're using ``.cxx`` for all the source files, can we change the extension for this file as well?

Reply:
The long term goal is to change all file names to follow the LLVM standard. In order to not make the pull requests to big, changing the names of only files that are otherwise touched is the way to go.